src/main/java/jnr/posix/LinuxFileStat64.java

```diff
@@ -10,21 +10,22 @@ public Layout(jnr.ffi.Runtime runtime) {
         }
 
         public final Signed64 st_dev = new Signed64();
-        public final Signed64 st_ino = new Signed64();
+        public final Unsigned64 st_ino = new Unsigned64();
         public final Signed64 st_nlink = new Signed64();
-        public final Signed32 st_mode = new Signed32();
-        public final Signed32 st_uid = new Signed32();
-        public final Signed32 st_gid = new Signed32();
+        public final Unsigned32 st_mode = new Unsigned32();
+        public final Unsigned32 st_uid = new Unsigned32();
+        public final Unsigned32 st_gid = new Unsigned32();
+        public final Signed32 __pad0 = new Signed32();
```

Comment:
It seems like these pads are in for word alignment purposes.  Our .java files are pretty old.  I wonder if they corrected alignment of stat in the last 10 years and ours are out of date with newer linuxes.  I see in FC25 the same pads.  I wish history was documented better in software.